armi/reactor/converters/uniformMesh.py

```diff
@@ -121,9 +135,14 @@ def initNewReactor(sourceReactor):
         # attributes are set when assemblies are added in coreDesign.construct(), however
         # since we skip that here, they never get set; therefore the need for the deepcopy.
         bp = copy.deepcopy(sourceReactor.blueprints)
-        newReactor = Reactor(sourceReactor.o.cs.caseTitle, bp)
+        newReactor = Reactor(sourceReactor.name, bp)
         coreDesign = bp.systemDesigns["core"]
-        coreDesign.construct(sourceReactor.o.cs, bp, newReactor, loadAssems=False)
+        if sourceReactor.o is None:
+            cs = settings.getMasterCs()
```

Comment:
What changed here?  We didn't used to have to support the case where `sourceReactor` was incomplete.

(If possible, I'm just trying to reduce the dependency on `setMasterCs()` and `getMasterCs()`, that's all.

Reply:
Added a comment in the code, but basically we didn't have a test case (integration or otherwise) that tested the use of the uniform mesh converter being chained together with the use of another geometry converter. For example, in most cases we have tested with `detailedAxialExpansion` set to False, but in the case where this is set to True (i.e., some benchmark problems we have) this results in the incoming reactor not having an operator attached. I don't think that it makes a ton of sense for a temporary reactor to _require_ an operator, so this is unfortunately needed at the moment until we can do a large refactor and decoupling of the reactor and the operator. I can make an issue to track this more explicitly, because it seems that the issue isn't present on Github.